R/misc.R

```diff
@@ -30,3 +30,182 @@
 "%not=%" <- function(a, b){
   ifelse(!is.na(a), a != b, F)
 }
+
+
+#' Check if a is greater than b
+#'
+#' @param a - a numeric variable present in the environment
```

Comment:
I was rethinking these functions, and I think that we should change everything for @param a to
"a variable ..." and not "a numeric variable ... " because these functions are existing to ensure a and b are both numeric. and a is not NA. So we should also include that both a should not be NA, or the result will be NA. 
Applicable to all